packages/libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -418,7 +442,7 @@ export class ChatsApi
    * @param params.currentUserId the user to act on behalf of
    * @returns the rpc object
    */
-  public async create(params: ChatCreateRequest) {
+  public async create(params: ChatCreateRequest): Promise<ChatCreateRPC> {
```

Comment:
nit: With the cast at the return site, technically this explicit type isn't needed and is inferred